src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,99 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int GROWTH_COEFFICIENT = 2;
+    private int size;
+    private Node<K, V>[] buckets;
 
     @Override
-    public void put(K key, V value) {
-
+    public int getSize() {
+        return size;
     }
 
     @Override
     public V getValue(K key) {
+        if (isEmpty()) {
+            return null;
+        }
```

Comment:
we dont need this check lets remove it (if hashMap is empty you wont get into while loop on 22 line and return null in 29)

Reply:
if i delete it it gets NPE, so i need it, or i don't know how to do it another way
![image](https://user-images.githubusercontent.com/85991807/168987790-43e86184-0704-47c9-ba97-a9232a74acdb.png)

Reply:
weird but now it works...